server/db/migrations/20171020144702_initial_schema.js

```diff
@@ -0,0 +1,41 @@
+exports.up = knex =>
```

Comment:
would be cool to have games table which stores all existing game stats,

players
joined_at

who scored
who won
etc

Reply:
Would this be for temporary use? If so, probably worth consulting with @OpenSourceCTF/engine requirements.